### ISSUE TITLE
✨ Add photo HDR capability

### ISSUE THREAD

**Issue by Balaji-Rahunathan** [cuvent/react-native-vision-camera#1473]
### What feature or enhancement are you suggesting?

Currently all formats shows only video HDR. It will be great if v3 has photo HDR capabilty

### What Platforms whould this feature/enhancement affect?

iOS

### Alternatives/Workarounds

I need this feature. Currently there are no alternatives or workarounds for this.

### Additional information

- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar feature requests in this repository](https://github.com/mrousavy/react-native-vision-camera/issues?q=is%3Aopen+is%3Aissue+label%3A%22✨+enhancement%22) and found none.

**Comment by mrousavy** [comment]
On iOS, photo HDR is absolutely non-trivial - looks like you need to capture multiple and fuse the images together yourself, basically re-invent the HDR algorithm. No idea why they don't provide that. See [Is HDR capture a thing in iOS?](https://www.reddit.com/r/iOSProgramming/comments/778ypz/is_hdr_capture_a_thing_in_ios/)

**Comment by mrousavy** [comment]
relevant: [Apple Docs: **Capturing a Bracketed Photo Sequence**](https://developer.apple.com/documentation/avfoundation/photo_capture/capturing_still_and_live_photos/capturing_a_bracketed_photo_sequence)

**Comment by mrousavy** [comment]
Hey! Sorry for the late reply and thank you for your issue report.
I was hyper-focused the entire winter to build the next major version of VisionCamera - and I think [VisionCamera V5](https://github.com/mrousavy/react-native-vision-camera/releases/tag/v5.0.0) fixes this! 🥳

Closing, but feel free to open a new issue if this still needs attention in V5! :)

**Comment by stevengoldberg** [comment]
Hi @mrousavy, in V5 I observe the same result as described in https://github.com/mrousavy/react-native-vision-camera/issues/3141 

On an iPhone 17 Pro, if I do:

```
const devices = useCameraDevices()
console.log('devices', devices)
```

I get an array of 10 devices, but every one has `supportsPhotoHDR: false`. Is this expected?

**Comment by mrousavy** [comment]
Yes, iOS (AVFoundation) does not expose Photo HDR as a separate knob. 

If you choose qualityPrioritization `quality` you _may_ get Smart HDR. This is not a specific knob but an implementation detail.
